src/uss/USSTree.ts

```diff
@@ -541,25 +543,35 @@ export class USSTree extends ZoweTreeProvider implements IZoweTree<IZoweUSSTreeN
      * Adds a single session to the USS tree
      *
      */
-    private async addSingleSession(profile: IProfileLoaded) {
-        if (profile) {
+    private async addSingleSession(profileLoaded: IProfileLoaded) {
+        if (profileLoaded) {
             // If session is already added, do nothing
-            if (this.mSessionNodes.find((tempNode) => tempNode.label.trim() === profile.name)) {
+            if (this.mSessionNodes.find((tempNode) => tempNode.label.trim() === profileLoaded.name)) {
                 return;
             }
-            // Uses loaded profile to create a session with the USS API
-            const session = ZoweExplorerApiRegister.getUssApi(profile).getSession();
+            let session;
+            try {
+                // Uses loaded profile to create a session with the USS API
+                session = await ZoweExplorerApiRegister.getCommonApi(profileLoaded).getValidSession(profileLoaded,
+                                                                                              profileLoaded.name,
+                                                                                              DefaultProfileManager.getInstance().getDefaultProfile("base"),
+                                                                                              false);
```

Comment:
you can't force call this .getValidSession as other extensions won't have this option yet.  It needs to be handled like Jelly handled the checkStatus